Tella/Data/Database/Common/SQLStatementBuilder.swift

```diff
@@ -30,7 +30,15 @@ class SQLiteStatementBuilder {
         return userVersion
         
     }
-    
+
+    func alterTable(tableName: String, column: String) {
```

Comment:
I think function name `alterTable` is not descriptive (the function is for adding a column)